src/components/FeeExplainer/FeeExplainer.tsx

```diff
@@ -313,12 +315,12 @@ export const FeeExplainer = () => {
 
   useEffect(() => {
     if (isLoading) return
-    if (currentFoxHoldings) setFoxHolding(Number(currentFoxHoldings))
+    if (currentFoxHoldings) setFoxHolding(Number(currentFoxHoldings) || 0)
```

Comment:
i'd have thought we can use `currentFoxHoldings` from `useGetVotingPowerQuery` directly without `const [foxHolding, setFoxHolding] = useState(Number(currentFoxHoldings) || 0)` since `currentFoxHoldings` will be undefined while it's loading

Reply:
i.e delete this whole useEffect?

Reply:
e3be5cc17dd58ad11ccf721ce1f5233fa4cd05e4